tenbilac/opt.py

```diff
@@ -44,15 +44,21 @@ def multnetbfgs(training, nepochs=10, maxiter_sum=200, maxiter_mult=200, gtol=1e
 	for epoch in range(nepochs):
 		logger.info("Epoch {}/{} starting".format(epoch, nepochs))
 		
-		training.set_paramslice(mode="sum")
-		bfgs(training, maxiter=maxiter_sum, gtol=gtol, **kwargs)
-		training.end()
```

Comment:
By the way, @kuntzer , why it training.end() called here ? Should we remove it and leave this to training.opt() ?